### ISSUE TITLE
Warn users that Facebook freezes titles after 50 actions

### ISSUE THREAD

**Issue by jonom** [jonom/silverstripe-share-care#2]
> You can always edit an object's properties, with two exceptions:
> 
> Title: Once 50 actions (likes, shares and comments) have been associated with an object, you won't be able to update its title
> Type: An object's type can't be changed once it's been used in a story

https://developers.facebook.com/docs/sharing/opengraph/using-objects

**Comment by sunnysideup** [comment]
Your module is great, but I reckon the ping connection with FB should be optional because it will slow down every save, AFAIK, and it does not add that much.... Otherwise great.  I am thinking about adding it to one of my own modules as a requirement.

**Comment by jonom** [comment]
@sunnysideup feel free to submit a PR for optional ping :) I think it should be quite useful though because Facebook caches that information pretty aggressively.

**Comment by sunnysideup** [comment]
I wonder what the harm is if fb has historical rather than current info
apart from when the is a mistake. Does fb cache page info from previous
shares ?

**Comment by jonom** [comment]
Yeah AFAIK FB only keeps one copy of OG data for each unique URL, and can hang on to it for a long time if you don't manually clear the cache. I couldn't find any specific info on Facebot crawl rates but I've seen some really out of date URL previews on Facebook before.

That request to Facebook to clear the cache only takes ~100ms for me so probably not causing a significant delay on page saving?

**Comment by sunnysideup** [comment]
Thank you for your info in this. That makes sense.

**Comment by jonom** [comment]
Seems like it's only an individual instance of a share that would get frozen so not a big deal
